docs-ref-services/other.md

```diff
@@ -21,7 +21,7 @@ ms.service: multiple
 
 ```bash
 pip install azure-common
-pip install azure-client
+pip install azure
```

Comment:
- We should never tell people to install explicitly "azure-common". It will be installed as needed by packages
- "azure" is a meta-package that installs both "client" library and "mgmt" library. If you are looking for "client-only" meta-package, that does not exist.